server/lib/filter.js

```diff
@@ -149,7 +149,12 @@ class FilterParser {
     const tableString = this._formatTableAlias(tableAlias);
 
     if (this._filters[filterKey]) {
-      const preparedStatement = `${tableString}${columnAlias} = ?`;
+      let valueString = '?';
+      if (filterKey.includes(DEFAULT_UUID_PARTIAL_KEY)) {
```

Comment:
There is a flag somewhere in this library that allows you to do this. I think in other controllers I've used the db (or util) parse UUIDs before passing the options to FilterParser. I'm not sure which one is better, as long as we clearly document that any key containing `uuid` will be converted it should be okay. 

I think this can be decided as the other controllers are converted to explicitly defining what should be filtered/ searched on.